cmd/mimir/help.txt.tmpl

```diff
@@ -311,6 +311,8 @@ Usage of ./cmd/mimir/mimir:
     	Other cluster members to join. Can be specified multiple times. It can be an IP, hostname or an entry specified in the DNS Service Discovery format.
   -modules
     	List available values that can be used as target.
+  -print-config-stderr
+    	Dump the entire Mimir config object to stderr in YAML format.
   -print.config
```

Comment:
Does this existing CLI flag help you? If not, we should probably deprecate it to avoid confusion.

Reply:
Wow! I don't know how I didn't find that. It does exactly what I need. Closing this PR 😅